src/components/Container/index.tsx

```diff
@@ -1,5 +1,29 @@
-import {Container, InnerContainer, OuterContainer} from "./styles";
+import { useState, useEffect } from 'react';
+import { InnerContainer, OuterContainer } from './styles';
 
-export {Container, InnerContainer, OuterContainer};
+export { Container, InnerContainer, OuterContainer };
 
-export default Container;
+export default function Container({
+  children,
+  backgroundColor
+}: {
+  children: any;
+  backgroundColor?: string;
+}) {
+  const [hasMounted, setHasMounted] = useState(false);
```

Comment:
I wonder if since we're now making this load on the client it's affecting how `TableOfContents/index.tsx` is working. The code in here looks a little old, maybe we just need to move the code snippet that is trying to scroll to the hash symbol down into the useEffect